forge/src/executor/inspector/cheatcodes/ext.rs

```diff
@@ -13,7 +13,8 @@ fn ffi(args: &[String]) -> Result<Bytes, Bytes> {
         .map_err(|err| err.to_string().encode())?
         .stdout;
     let output = unsafe { std::str::from_utf8_unchecked(&output) };
-    let decoded = hex::decode(&output.trim()[2..]).map_err(|err| err.to_string().encode())?;
+    let decoded = hex::decode(&output.strip_prefix("0x").unwrap_or(output))
```

Comment:
i think the trim is still needed?
```suggestion
    let decoded = hex::decode(&output.trim().strip_prefix("0x").unwrap_or(output))
```